common/OpTestLpar.py

```diff
@@ -685,3 +685,79 @@ def lpar_hexdump(self, i_dev):
             l_msg = "hexdump failed for device %s" % i_dev
             print l_msg
             raise OpTestError(l_msg)
+
+    ##
+    # @brief It will clone latest skiboot git repository in i_dir directory
+    #
+    # @param i_dir @type string: directory where skiboot source will be cloned
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def lpar_clone_skiboot_source(self, i_dir):
```

Comment:
@geissonator please correct me if I'm wrong here.

We can't install anything onto the system using the testcases. The user must install the required tools/software before running the tests and specify the directory before using it.